tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -1028,6 +1028,7 @@ cc_library(
         "//tensorflow/compiler/xla/service:call_inliner",
         "//tensorflow/compiler/xla/service:conditional_simplifier",
         "//tensorflow/compiler/xla/service:convolution_group_converter",
```

Comment:
I think this dependency can be removed because we don't run ConvolutionGroupConverter anymore.